signxml/xades/__init__.py

```diff
@@ -326,12 +340,9 @@ def _generate_xades_ssp_elements(self, options_struct):
                 encoding of one DER-encoded instance of type IssuerSerial type
                 defined in IETF RFC 5035 [17].
                 """
-                raise NotImplementedError(
-                    "Please make a PR if you know how to obtain in python a "
-                    "'DER-encoded instance of type IssuerSerial type defined in "
-                    "IETF RFC 5035'")
                 serial_element = XADES132.IssuerSerialV2(
-                    # TODO implement, wtf?
+                    DS.X09IssuerName(issuer_string(cert)),
+                    DS.X509SerialNumber(b64encode(_der.encode_der_integer(cert.get_serial_number())))
```

Comment:
The spec is something like:
`XADES132.IssuerSerialV2(b64encode(_der.encode...(asn_obj)))`

Reply:
IMO must defined from `DS.X509SerialNumber` element wil double check

Reply:
fixed